modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -280,13 +348,18 @@ private List<Parameter> _getParameters(
 		for (Parameter parameter : operation.getParameters()) {
 			String parameterName = parameter.getName();
 
-			if (Objects.equals(parameterName, "objectRelationshipName")) {
+			if (Objects.equals(parameterName, "objectActionName") ||
+				Objects.equals(parameterName, "objectRelationshipName")) {
+
 				continue;
 			}
 
-			if (Objects.equals(parameterName, "currentObjectEntryId")) {
+			if (Objects.equals(parameterName, "currentObjectEntryId") ||
+				Objects.equals(parameterName, "objectEntryId")) {
```

Comment:
Hi!
This validation is not needed as we are changing the `objectEntryId` in the `openAPISchemaFilter` provided in the creation of the initial schema.

Reply:
I updated the code removing this validation.

Thanks!